src/components/manage/Widgets/__snapshots__/FileWidget.test.jsx.snap

```diff
@@ -1,81 +1,39 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`renders a file widget component 1`] = `
-Array [
+<div>
   <div
-    className="inline field"
+    class="inline field"
   >
     <div
-      className="ui grid"
+      class="ui grid"
     >
       <div
-        className="stretched row"
+        class="stretched row"
       >
         <div
-          className="four wide column"
+          class="four wide column"
         >
           <div
-            className="wrapper"
+            class="wrapper"
           >
             <label
-              htmlFor="field-my-field"
+              for="field-my-field"
             >
               My field
             </label>
           </div>
         </div>
         <div
-          className="eight wide column"
+          class="eight wide column"
         >
           <div
-            className="file-widget-dropzone"
```

Comment:
@sneridagh is the fileWidget component completely gone in this test snapshot?

Reply:
Something weird is going on in this test, we have to figure it out. Previously, the test output was an array (!?!?) which is weird. I changed to use @testing-library and it behaved better, but I overlooked that the widget was gone. It's a matter of timing issues in testing rendering... the widget is there for sure, if not, cypress won't pass...